src/main/java/filter/expression/Qualifier.java

```diff
@@ -1031,11 +1031,9 @@ private Qualifier convertMilestoneAliasQualifier(List<TurboMilestone> allMilesto
             }
         }
 
-        // clamp
-        if (currentIndex >= allMilestones.size()) {
-            currentIndex = allMilestones.size() - 1;
-        } else if (currentIndex < 0) {
-            currentIndex = 0;
+        // if out of milestone range, don't convert alias
+        if (currentIndex >= allMilestones.size() || currentIndex < 0) {
+            return this;
```

Comment:
I guess you could call it the null (or false, if we don't want to overload the term) qualifier. Implementing it is mostly just figuring out a way to differentiate it from others, then adding it to isSatisfiedBy. I don't think we need to expose it to users.